_posts/2021-02-01-coalesce-dbt-summary.markdown

```diff
@@ -0,0 +1,288 @@
+
+On a more technical level, I took away from many talks the success teams had encountered by consolidating the knowledge and business logic of data domain and insight generation within their DBT projects, and how this had promoted more transparency and trust between consumers and producers / curators of data.
+
+
+### The Right Testing Matters
+
+There were a couple of great talks about testing in Coalesce 2020; beyond introducing tools and technologies, they more importantly offered:
+
+* The importance of testing as part of [SRE](https://sre.google/sre-book/table-of-contents/) obligations and meeting service level agreements
+* Different types of testing, e.g. modelling regression vs. data regression, including periodic factors
+* Comparing the value of testing against the cost of troubleshooting queries from consumers
+* The use of testing as a form of documentation about expectations and subject matter expertise
+
+One important reminder I took away was, ensure to test the right thing: avoid vanity tests and appreciate that needs and expectations, as well as data, change over time - and therefore, so should your tests; even if it means deprecation.
+
+
+### Trust 
+
+Perhaps the most enduring theme throughout the presentations was trust:
+
+* Why is it important?
+* What happens when we don't have it?
+* How can we build it?
+* How can we retain it?
+
+Trust exists in many contexts; some examples were how teams build trust amongst themselves with testing and documentation; how they maintain it with peer teams using SLAs, relationships and communication, and good engineering practice; how they build it with stakeholders and decision makers, to boost their standing and perception as a critical partner within an organisation.
+
+I sensed in many presentations the burden within data teams that their work should reflect the most true view possible of their organisation. The concept of trust featured heavily in many of their endeavours. All of the following stood out as important assets in building and maintaining that trust.
+
+- Engagement and Discoverability of Data
+- Operational Resiliency of Data Services and Products
+- Verifiable Data Quality In Decision Making
+- Knowledge Formation and Consensus Building
+- Finding and Partnering with the right stakeholders
+
+In researching this article, I found a useful EY article by Amy Brachio that discusses [relationships between risk and trust](https://www.ey.com/en_au/wef/four-ways-to-build-a-relationship-between-risk-and-trust): it summarises many of the subtle dynamics highlighted in various talks from Coalesce 2020. For example, when considering risks around stakeholder engagement - in particular, that you are working with the right stakeholders - Coalesce 2020 taught me that if you are not being challenged on your assertions by your stakeholders, then perhaps they are not fully representative of the customers you believe you serve.
```

Comment:
Again clunky last sentence.

How about : "For example, when considering the risk that you are not working with the right stakeholders - if you are not being challenged on your assertions by your stakeholders, then perhaps they are not fully representative of the customers you serve."

Reply:
i don't think yours is any better but i take the point and will give it some love